model/src/main/java/com/basistech/rosette/dm/AttributeKey.java

```diff
@@ -52,8 +52,8 @@
     TRANSLATED_TOKENS("translatedTokens"),
     TOPIC_RESULTS("topicResults"),
 
-    CONCEPT("concepts"),
-    KEYPHRASE("keyphrases");
+    CONCEPT("concept"),
```

Comment:
But isn't it the same for "token", "sentence", etc. (but not for "entities", I agree). I agree it sounds weird, I just felt it's more consistent with other fields in this enum. I'm fine with changing it back, and I'll ping Hannah on this.